packages/feedback/src/environments/environment.dev.ts

```diff
@@ -0,0 +1,17 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=dev` then `environment.dev.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: false,
+  name: '',
+  api: '',
+  graphqlAPI: '',
+  client: '',
+  authIssuer: '',
+  matomoSiteID: null, // Site ID from Piwik Analytics
```

Comment:
No need for Matomo, serviceNow, authIssuer here.
Those values are not being used anywhere in the application.

Reply:
fixed it.